src/integrators/low_order_rk_integrators.jl

```diff
@@ -284,22 +237,21 @@ end
   integrator.k[integrator.kshortsize] = integrator.fsallast
 end
 
-#=
-@inline function perform_step!(integrator,cache::Tsit5ConstantCache,f=integrator.f)
+@inline @muladd function perform_step!(integrator,cache::Tsit5ConstantCache,f=integrator.f)
   @unpack t,dt,uprev,u,k = integrator
   @unpack c1,c2,c3,c4,c5,c6,a21,a31,a32,a41,a42,a43,a51,a52,a53,a54,a61,a62,a63,a64,a65,a71,a72,a73,a74,a75,a76,b1,b2,b3,b4,b5,b6,b7 = cache
   k1 = integrator.fsalfirst
   a = dt*a21
-  k2 = f(@muladd(t+c1*dt),@.(@muladd(uprev+a*k1)))
-  k3 = f(@muladd(t+c2*dt),@.(@muladd(uprev+dt*(a31*k1+a32*k2))))
-  k4 = f(@muladd(t+c3*dt),@.(@muladd(uprev+dt*(a41*k1+a42*k2+a43*k3))))
-  k5 = f(@muladd(t+c4*dt),@.(@muladd(uprev+dt*(a51*k1+a52*k2+a53*k3+a54*k4))))
-  k6 = f(t+dt,@.(@muladd(uprev+dt*(a61*k1+a62*k2+a63*k3+a64*k4+a65*k5))))
-  u = @. @muladd uprev+dt*(a71*k1+a72*k2+a73*k3+a74*k4+a75*k5+a76*k6)
+  k2 = f(t+c1*dt, @. uprev+a*k1)
+  k3 = f(t+c2*dt, @. uprev+dt*(a31*k1+a32*k2))
+  k4 = f(t+c3*dt, @. uprev+dt*(a41*k1+a42*k2+a43*k3))
+  k5 = f(t+c4*dt, @. uprev+dt*(a51*k1+a52*k2+a53*k3+a54*k4))
+  k6 = f(t+dt, @. uprev+dt*(a61*k1+a62*k2+a63*k3+a64*k4+a65*k5))
```

Comment:
are the `@.`s in these spots okay for performance? It should be fine in the long run since I'm hoping to get that fixed so we should stick with it for now (and usage with actual arrays should just be inplace anyways), but I'd like to know if you've checked.